modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/screen/RequestScreen.js

```diff
@@ -265,37 +265,44 @@ class RequestScreen extends Screen {
 				method: httpMethod,
 				mode: 'cors',
 				redirect: 'follow',
-				referrer: 'client'
+				referrer: 'client',
 			})
-				.then(resp => {
+				.then((resp) => {
 					this.assertValidResponseStatusCode(resp.status);
 
 					this.setResponse(resp);
 
 					return resp.clone().text();
 				})
-				.then(text => {
-					if (httpMethod === RequestScreen.GET && this.isCacheable()) {
+				.then((text) => {
+					if (
+						httpMethod === RequestScreen.GET &&
+						this.isCacheable()
+					) {
 						this.addCache(text);
 					}
 
 					return text;
 				}),
 			new Promise((_, reject) => {
-				setTimeout(() => reject(new Error(errors.REQUEST_TIMEOUT)) , this.timeout);
-			})
+				setTimeout(
+					() => reject(new Error(errors.REQUEST_TIMEOUT)),
```

Comment:
Ah, and I see you fixed the whitespace here (I am commenting on this commit-by-commit, so I am not seeing the final state until I get to the end).